src/components/slider/slider.scss

```diff
@@ -76,6 +84,11 @@ $colorGrey: #999999;
     top: pxToRem(-16);
     transform: none;
     width: pxToRem(40);
+
+    .mdc-slider.mdc-slider--disabled & {
+        background: var(--mdc-slider-bg-color-behind-component);
+        background: linear-gradient(90deg, rgba(255,255,255,0) 0%, var(--mdc-slider-bg-color-behind-component) 15%, var(--mdc-slider-bg-color-behind-component) 85%, rgba(255,255,255,0) 100%);
```

Comment:
It's inevitable that there will be positions where the slider thumb will partially cover one of the min- and max-labels, so I added a small gradient to make it a little nicer looking when that happens (only applies when the slider is disabled).

Hot or Not?

Reply:
in my opinion it's rather overkill. And maybe I need to let my eyesight checked, but I really not see much of a difference especially with a white background. I'm more bothered by the fact, that the pins background is not just the pin but also the dot on the line. You can't really tell when it's white, but it looks kinda weird with a color.
![image](https://user-images.githubusercontent.com/10260886/64171724-a22d5c80-ce53-11e9-9b83-1d64288a6fed.png)
anyhow I think I've been at that point and you can't set the background just to the pin :roll_eyes:

Reply:
Yeah, but that's because you're not supposed to set the background colour to anything but the background colour. If you change the background colour that the component rests on as well, you'll see why there's a ring around the dot:

master:

![image](https://user-images.githubusercontent.com/154725/64175602-f2102180-ce5b-11e9-8cb8-566b6c77d81e.png)

pr:

![image](https://user-images.githubusercontent.com/154725/64175352-6bf3db00-ce5b-11e9-9462-5ac5be48fce1.png)

Reply:
The reason for the gradient is different.

Without gradient:

![image](https://user-images.githubusercontent.com/154725/64176135-0a347080-ce5d-11e9-9e64-c5c7d12eeacb.png) ![image](https://user-images.githubusercontent.com/154725/64176156-17e9f600-ce5d-11e9-9ffb-449892f664cd.png)

With gradient:

![image](https://user-images.githubusercontent.com/154725/64176097-fdb01800-ce5c-11e9-80c7-372bae6e7d76.png) ![image](https://user-images.githubusercontent.com/154725/64176082-f2f58300-ce5c-11e9-830d-501294e9b2c2.png)

Reply:
okay, you convinced me, it's better with the gradient. But maybe we shouldn't have the `--slider-bg-color-behind-component` variable then and just set it to white?

Reply:
It defaults to white, but the variable is there in case anyone ever puts it on a different background colour. They probably shouldn't in the webclient, but maybe in another application. Go wants to use lime-elements for example.